test/Microsoft.TestTemplates.Acceptance.Tests/DotnetCoreTemplateTests.cs

```diff
@@ -1,24 +1,21 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-namespace Microsoft.TestTemplates.Acceptance.Tests
-{
+namespace Microsoft.TestTemplates.Acceptance.Tests {
     using Microsoft.VisualStudio.TestTools.UnitTesting;
     using System.Collections.Generic;
     using System.IO;
+    using System.Text.RegularExpressions;
 
     [TestClass]
-    public class DotnetCoreTemplateTests : AcceptanceTestBase
-    {
+    public class DotnetCoreTemplateTests : AcceptanceTestBase {
         /// <summary>
         /// The net core versions for which templates are present
         /// </summary>
         private static string[] netCoreVersions = {
             // refer to https://dotnet.microsoft.com/download/dotnet-core
             // for a list of supported dotnet versions and only include the ones
             // that are not end-of-life
-            "3.1",
-            "5.0",
             "6.0",
```

Comment:
@Evangelink here I'm removing eof versions but I've updated also old template, but we're testing only non eol, what you think? should we:
* test eol
* do not add new prop to the eol
* keep as-is

Reply:
removed eol for now